pkg/autodiscovery/listeners/README.md

```diff
@@ -13,7 +13,7 @@ Services can only be Docker containers for now.
 
 ### `DockerListener`
 
-`DockerListener` first gets current running containers and send these to `ConfigResolver`. Then it starts listening on the Docker event API for container activity and pass by `Services` mentioned in start/stop events to `ConfigResolver` through the corresponding channel.
+`DockerListener` first gets current running containers and send these to the `AutoConfig`. Then it starts listening on the Docker event API for container activity and pass by `Services` mentioned in start/stop events to the `AutoConfig` through the corresponding channel.
```

Comment:
`AutoConfig` or `Autodiscovery` ? I think we stated that we should use one or the other

Reply:
We should use `Autodiscovery` for the whole mechanism but here I'm talking about the `AutoConfig` struct itself (that we should rename as well at some point)